string camera_info

---

float64[] location